### DISCUSSION TITLE
Issues in m4 "git" build needs patches in gnulib, which is cloned during bootstrap

### DISCUSSION THREAD

**Discussion by HarithaIBM** [zopencommunity/meta#216] [General]
@MikeFultonDev @IgorTodorovskiIBM 

I have the m4 git build running successfully on zoscan2b machine. But, this needs fixes in gnulib which gets cloned as part of bootstrap step.
How do i apply this patch, which gets applied after bootstrap step?

While Igor and I, were discussing a thought which came was that - we come up with a new script which applies patches after bootstrap step or specify a separate gnulib src folder to pickup during bootstrap?

@MikeFultonDev Any thoughts here, please! Need help to conclude!

Thank you.

**Comment by MikeFultonDev** [comment]
Great progress! 
can we use gnulibport to work on the patches and then upstream them? It will take longer but we don’t have a hard timeline on it. 
I had good luck bringing patches forward before for gnulib.

**Comment by MikeFultonDev** [comment]
closing this discussion off and opened up an issue here: https://github.com/ZOSOpenTools/m4port/issues/30
